src/tools/docker-to-docker-compose/docker-to-docker-compose.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <n-card>
+    <n-input
+      v-model:value="dockerRun"
+      style="font-family: monospace"
+      type="textarea"
+      placeholder="Your docker run command to convert..."
+      rows="3"
+    />
+
+    <n-divider />
+
+    <n-input
+      style="font-family: monospace"
+      :value="dockerCompose"
+      type="textarea"
+      placeholder="Converted to docker-compose file"
+      :autosize="{ minRows: 1 }"
+      readonly
+      autocomplete="off"
+      autocorrect="off"
+      autocapitalize="off"
+      spellcheck="false"
```

Comment:
Prefer using `textarea-copyable` instead of `input` with language `yaml` for syntaxe highlight, see `src/tools/html-wysiwyg-editor/html-wysiwyg-editor.vue` if needed

Reply:
Done. Removed the copy button because the textarea-copyable already has the copy feature included.